src/schema/users.ts

```diff
@@ -567,6 +648,44 @@ export const resolvers: IResolvers<any, Context> = {
         )}.png?r=${randomStr}`,
       };
     },
+    updateUserProfile: async (
+      _,
+      { data }: GQLUserParameters,
+      ctx,
+    ): Promise<GQLUser> => {
+      const repo = ctx.con.getRepository(User);
+      const user = await repo.findOne({ id: ctx.userId });
+
+      if (!user) {
+        throw new AuthenticationError(
```

Comment:
Let's just throw a forbidden error (or whatever is thrown when the user is not authenticated) with no explanation in this case. This will prevent leaking our database in extreme cases

Reply:
Oh, cool. It is `UNAUTHENTICATED` which I think is Authentication Error. Let me remove the error message.